tools/fleetd-linux/docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+---
+x-default-settings:
+  environment: &default-environment
+    ENROLL_SECRET: "${ENROLL_SECRET:?ENROLL_SECRET must be set for server authentication}"
+  ulimits: &default-ulimits
+    core:
+      hard: 1000000000
+      soft: 1000000000
+  network_mode: &default-network-mode bridge
+  extra_hosts: &extra-hosts
+    - "host.docker.internal:host-gateway"
+  platform: &default-platform linux/x86_64
+
+services:
+  ubuntu24.04-fleetd:
+    image: "fleetd-ubuntu-24.04"
+    privileged: true
```

Comment:
Why do we need privileged here?

Reply:
I see these (osquery is running as `root`, but we still get those if `privileged` is not `true`):
```sh
debian12.8-fleetd-1       | I1227 17:33:32.143419    49 processes.cpp:510] Cannot read /proc/50/io (is osquery running as root?)
amazonlinux2023-fleetd-1  | I1227 17:33:33.037750    49 processes.cpp:510] Cannot read /proc/50/io (is osquery running as root?)
redhat9.5-fleetd-1        | I1227 17:33:33.103796    48 processes.cpp:510] Cannot read /proc/49/io (is osquery running as root?)
fedora41-fleetd-1         | I1227 17:33:33.253508    49 processes.cpp:510] Cannot read /proc/50/io (is osquery running as root?)
ubuntu24.04-fleetd-1      | I1227 17:33:33.349644    49 processes.cpp:510] Cannot read /proc/50/io (is osquery running as root?)
```

Reply:
Got it. Fair enough. Betting it's based on which queries need to run.